index.html

```diff
@@ -103,9 +103,42 @@ <h3 class="title-post">Multi-Post Stories</h3>
                                 <li class="tag-4">Ruby</li>
                             </ul>
                         </div>
-                        <button class="button-p-main" type="button">See Project</button>
+                        <button data-modal-target="#modal" class="button-p-main" type="button">See Project</button>
                     </div>
-
+                    <div class="modal" id="modal">
+                        <div class="modal-header">
+                          <div class="title">Keeping track of hundreds  of components website</div>
+                          <button data-close-button class="close-button">&times;</button>
+                        </div>
+                        <div class="modal-body">
+                            <div class="language-tags-model language-tags">
+                                <ul class="tags-model">
+                                    <li class="model-tag tag-1">css</li>
+                                    <li class="model-tag tag-2">html</li>
+                                    <li class="model-tag-b tag-3">bootstrap</li>
+                                    <li class="model-tag tag-4">Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="model-main-project">
+                                <div class="model-Img-Placeholder">
+                                </div>
+                                <div class="model-project-left">
```

Comment:
- [ ] The image on the modal is floating out of its parent div on a mobile screen. Kindly work on your styles to make the image on the mobile popup match the Figma design. Please work on your styles. 👍 

**See your design**
![image](https://user-images.githubusercontent.com/50941074/169157715-d9052eec-a153-4f57-835a-4b8555e2a218.png)

**See the Figma design**
![image](https://user-images.githubusercontent.com/50941074/169157891-841ea904-c98b-4ead-9339-6385ad1ec30d.png)


- [ ] I also noticed that on the mobile screen, the See Project button is missing on all six projects. The See Project button should be there in order for a user to click and access the modal window for that project and see its details. You're only implementing that for the first project. Kindly make all your projects have that button and when clicked, the details for that particular project are fetched from the projects array and displayed its data in the modal window. 👍👍 

**See your design**
![image](https://user-images.githubusercontent.com/50941074/169158922-7a6ca638-c449-458a-a4e9-5ed79bd91811.png)

**The Figma design**
![image](https://user-images.githubusercontent.com/50941074/169159014-684ced79-b476-4cb7-a675-1d2925486e05.png)